src/network/networkd-ndisc.h

```diff
@@ -1,13 +1,11 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
-#include "conf-parser.h"
-#include "dns-resolver-internal.h"
-#include "time-util.h"
+#include <netinet/in.h>
 
-typedef struct Address Address;
-typedef struct Link Link;
-typedef struct Network Network;
+#include "dns-resolver-internal.h"
+#include "networkd-forward.h"
+#include "memory-util.h"
```

Comment:
Maybe better to make `NDISC_DNSSL_DOMAIN` non-static? Then we can drop memory-util.h here?